packages/playwright-config-commons-ui/index.js

```diff
@@ -0,0 +1,35 @@
+const { devices } = require("@playwright/test");
+
+const config = {
+  timeout: 30 * 1000,
+  testMatch: /.*\.spec.js/,
+  reporter: "html",
+  use: {
+    actionTimeout: 0,
+    headless: false,
+    viewport: { width: 1280, height: 720 },
+    trace: "on-first-retry",
+  },
+  projects: [
+    {
+      name: "chromium",
+      use: {
+        ...devices["Desktop Chrome"],
+      },
+    },
+    {
+      name: "firefox",
+      use: {
+        ...devices["Desktop Firefox"],
+      },
+    },
+    {
+      name: "webkit",
+      use: {
+        ...devices["Desktop Safari"],
+      },
+    },
```

Comment:
```suggestion
    {
      name: "webkit",
      use: {
        ...devices["Desktop Safari"],
      },
    },
```
Not sure this will run on Ubuntu

Reply:
@kilemensi There seems to be a fix here https://github.com/microsoft/playwright-test/issues/244

Reply:
Just because I'm the only one using Ubuntu you guys want to throw me under the bus like [this](https://github.com/microsoft/playwright/issues/4236#issuecomment-855282593)?

What about if we want to run these tests in GitHub Action? We'll have to manually install all those packages as well?